docs/preview/03-Features/powershell/azure-logic-apps.md

```diff
@@ -33,6 +33,39 @@ PS> Cancel-AzLogicAppRuns `
 # Successfully cancelled all running instances for the Azure Logic App 'rcv-shopping-order-sftp' in resource group 'rg-common-dev'
 ```
 
+## Resubmitting failed instances for an Azure Logic App
+
+Use this script to resubmit failed instances for a specific Azure Logic App. 
```

Comment:
I'm not a Logic App expert, so excuse my ignorance: is this description enough to know what 'resubmitting a Logic App' would result in? What consequences or so?

Reply:
Yes I think so 😉 
To be sure I checked this description with a colleague with some Logic App experience and he understood as well. A bit of a small sample size but I think we can leave it like it is now.

Reply:
Yes correct, I rewrote it a little bit so it is more clear.